packages/@n8n/config/src/configs/database.ts

```diff
@@ -0,0 +1,159 @@
+import { Config, FromEnv, Nested } from '../decorators';
+
+@Config
+class LoggingConfig {
+	/** Typeorm logging enabled flag. @default false */
+	@FromEnv('DB_LOGGING_ENABLED')
```

Comment:
Subjective, I'd prefer shorter syntax:

```suggestion
	@Env('DB_LOGGING_ENABLED')
```